common/supplemental/supplementalData.xml

```diff
@@ -1628,7 +1629,7 @@ XXX Code for transations where no currency is involved
 		<language type="gos" scripts="Latn"/>
 		<language type="got" scripts="Goth" alt="secondary"/>
 		<language type="grb" scripts="Latn"/>
-		<language type="grc" scripts="Cprt Grek Linb" alt="secondary"/>
+		<language type="grc" scripts="Cprt Grek" alt="secondary"/>
```

Comment:
should this be primary Grek and secondary Cprt (which as far as i can tell isn't even 100% historically certain)

Reply:
The list here is alphabetical not ordered by frequency. Another reason I want to redo tag/attribute hierarchy here.

I'll look into removing the auto-alphabetical ordering for the script attribute. It may not be too tricky.

Reply:
Yea, I cannot find the right piece of code to do that.

Now that UTW's done I can work more on my proposal to redo this section.

Reply:
I thought that was the job of `language_script.tsv` but i really don't know any more-

yeah

```
grc	Ancient Greek	secondary	Cprt	Cypriot
grc	Ancient Greek	secondary	Grek	Greek
grc	Ancient Greek	secondary	Linb	Linear B
```

@macchiati why is Grek secondary here with no primary? ^

https://github.com/unicode-org/cldr/blob/4898ee6b1283473f70f50de2d4bacad802993746/tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/language_script.tsv#L284